recipes/r-rvcg/meta.yaml

```diff
@@ -0,0 +1,130 @@
+{% set version = '0.22.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rvcg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/Rvcg_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/Rvcg/Rvcg_{{ version }}.tar.gz
+  sha256: dbaa2e76fd5f7e3d09f9cc326eea2b225be1be49c4f21b403d0bd3c3f114399b
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  
+  missing_dso_whitelist:
+    - '*/R.dll'
```

Comment:
I've never seen this before. Not saying it needs to change, but is there a precedent?

Reply:
Please check here:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=660876&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&l=3629

Reply:
Ah great, that makes sense. I tried compiling on my windows machine and Rvcg loads normally with R.dll whitelisted. I guess this is fine? (If so, can I mark this conversation as resolved?

Reply:
Yes, but I think I marked it already. ✅ And thanks for testing locally!